flow-typed/Collections.js

```diff
@@ -2,6 +2,7 @@ declare type Collection = {
   id: string,
```

Comment:
1. Create a new playlist from `/$/playlists` called `tmp`
2. Go back to homepage, and select `Add to tmp` from any claim.
3. Publish the claim, only changing channel to Anon and bid to 0.0001.

The claimID looks bad:

<img width="1268" alt="image" src="https://user-images.githubusercontent.com/64950861/186693979-f5bf4c14-0da0-4173-8123-b18c11a0765d.png">